### PR TITLE
Create baseTheme

### PR DESCRIPTION
**Pull request by lulen11**
Hey @kbardi I've setup the baseTheme in this PR but haven't implemented any of the styles just yet. Could you help check over it and see if anything else needs to be done?

### CODE REVIEW

**Comment by lulen11** [comment]
hey @kbardi have you had a chance to look at this one yet? :D

**Comment by lulen11** [comment]
Just a note to say that this is referring to the 'wireframe' theme we keep talking about implementing

**Comment by lulen11** [comment]
NOTE: Not ready to be merged, should have opened it as a Draft PR

**Comment by lulen11** [comment]
Hi @rin @davidpaley @kbardi, 

Could you please review this PR? The `base` theme has been setup and I've checked the components are all setup to cater for themes. There are many components that need alterations and updates but I want to do them in separate branches because the adjustments don't solely relate to one theme, rather they'll be adjusted across all the themes. I will create separate issues for them. But for now would be great if I could merge this back in with `master` and then branch off to work on the components (such as, `select`, `section`, `footer`... there are more though) individually.

**Comment by rin** [comment]
Things to do:
- [ ] Update the tests (I guess one of us could do that – I'll put it on my "to do if I get to it today" list)
- [ ] Resolve merge conflicts

Question, if the baseTheme is the default theme, do we need to change something in Portal when using blueprint and setting the purple theme? @lulen11

**Comment by rin** [comment]
Some things I noticed …
Input and Textbox/Textarea should at least have a border so that you can see then on a white background?

Some components that look broken in the Styleguide, but where I’m not sure if it happened in this PR:
- Loading (all black?) - this works in master
- Checkbox (weird colours in example?)
- Country Flag (not visible) - this is also broken in master, so not an issue of this PR
- Country Flag Selector  - this is also broken in master, so not an issue of this PR

@lulen11

**Comment by lulen11** [comment]
Hey @rin thanks so much for reviewing!! 

With the tests & merge conflicts, @kbardi has mentioned he's aware of these and I believe it was because of something he wanted to update on the `master` branch? Maybe I'm mixing up different broken things there but none-the-less I have been ignoring them because (I don't understand them) he's looking into them 😅 

> Question, if the baseTheme is the default theme, do we need to change something in Portal when using blueprint and setting the purple theme? @lulen11

I think @kbardi could help answer this question too but... Correct me if I'm wrong @kbardi... when we're using blueprint in different projects, we will define which theme they are to use. Eg. Yes, Portal would use the `portal` theme. (for now, Portal would use `portal` theme but I will be rethinking the theming on Blueprint very soon and I might define a `product` theme and a `website` theme because we would have styling for our products such as Theatre and Portal which would look different to the external website styles... But as I said, still am yet to rethink this... Just a note) THOUGH, the `base` theme is more like a foundations, wireframe, type theme. It's greyscale and will be what we use as a base when building out our interfaces. Does that make sense @rin ?

> Input and Textbox/Textarea should at least have a border so that you can see then on a white background?

I'm not suuuure that these should have a border as I believe they will always be setup to be on a background colour. Still, most style things are yet to be decided with the rebrand that @charliemckenzie is working on. 

> * Loading (all black?) - this works in master
> * Checkbox (weird colours in example?)

Both of these look different (greyscale) because I've set blueprint to show components as `base` theme while I setup all the components to work in the `base` theme... If we adjust the theme to be `portal` or `purple` they should go back to their colourful ways. 

> * Country Flag (not visible) - this is also broken in master, so not an issue of this PR
> * Country Flag Selector  - this is also broken in master, so not an issue of this PR

Yep both of these are already broken and will be addressed in separate branches 😄

File: src/lib/components/progressBar/progressBar.module.scss [hunk 1]
```diff
@@ -1,9 +1,9 @@
-@import '../../styles/variables/variables.module.scss';
-@import '../../styles/themes/themes.module.scss';
+@import "../../styles/variables/variables.module.scss";
+@import "../../styles/themes/themes.module.scss";
 
 .progress {
   overflow: hidden;
-  background-color: $medium-grey;
+  background-color: $grey-60;
```
Comment (mischacolley) [inline src/lib/components/progressBar/progressBar.module.scss]: Shouldn't this be in a themify block?

File: src/lib/components/progressBar/progressBar.module.scss [hunk 2]
```diff
@@ -15,7 +15,7 @@
   font-size: 12px;
```
Comment (mischacolley) [inline src/lib/components/progressBar/progressBar.module.scss]: Shouldn't this be in a themify block too?

File: src/lib/components/table/table.module.scss [hunk 3]
```diff
@@ -33,6 +37,10 @@
   text-align: center;
   font-size: 0.9rem;
   border-bottom: 1px solid $grey-lavender;
```
Comment (mischacolley) [inline src/lib/components/table/table.module.scss]: Shouldn't this be in a themify block?

File: src/lib/components/progressBar/progressBar.module.scss (hunk 1 above)
Comment (lulen11) [reply src/lib/components/progressBar/progressBar.module.scss]: yes! fixed!

File: src/lib/components/progressBar/progressBar.module.scss (hunk 2 above)
Comment (lulen11) [reply src/lib/components/progressBar/progressBar.module.scss]: yes! fixed!

File: src/lib/components/table/table.module.scss (hunk 3 above)
Comment (lulen11) [reply src/lib/components/table/table.module.scss]: yes! fixed!

**Comment by lulen11** [comment]
@kbardi @davidpaley @rin I'm looking 👀 for one more positive review ploiiiise :)

**Pull request merged**